components/create_post/index.js

```diff
@@ -46,6 +48,8 @@ function mapStateToProps() {
         const getCommentCountForPost = makeGetCommentCountForPost();
         const latestReplyablePostId = getLatestReplyablePostId(state);
         const currentChannelMembersCount = getCurrentChannelStats(state) ? getCurrentChannelStats(state).member_count : 1;
+        const enableTutorial = config.EnableTutorial === 'true';
+        const tutorialStep = parseInt(get(state, Preferences.TUTORIAL_STEP, getCurrentUserId(state), TutorialSteps.FINISHED), 10);
```

Comment:
I was going to recommend using the `getInt` selector, but there actually isn't one. We should add that at some point to be like the `getBool` one that we do have